agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryItemExporter.java

```diff
@@ -126,6 +166,53 @@ CompletableResultCode internalSendByConnectionString(
     return telemetryPipeline.send(byteBuffers, connectionString, listener);
   }
 
+  private TelemetryItem createOtelResourceMetric(
+      String connectionString, Map<String, String> existingTags) {
+    MetricTelemetryBuilder builder = MetricTelemetryBuilder.create(_OTELRESOURCE_, 0);
+    builder.setConnectionString(connectionString);
```

Comment:
I don't think we need to set `connectionString` (since we have already grouped the telemetry by connection string at this point

Reply:
need to stamp ikey into _otelresource_ custom metric